Doc/library/pathlib.rst

```diff
@@ -1054,6 +1054,12 @@ call fails (for example because the path doesn't exist).
    use :func:`Path.rmdir` instead.
 
 
+.. method:: Path.link(target)
```

Comment:
Since we already have `symlink_to`, this method should probably be called `link_to`.